daprdocs/content/en/operations/performance-and-scalability/perf-service-invocation.md

```diff
@@ -0,0 +1,89 @@
+---
+type: docs
+title: "Service invocation performance"
+linkTitle: "Service invocation performance"
+weight: 10000
+description: ""
+---
+This article provides service invocation API performance benchmarks and resource utilization for the components needed to run Dapr in different hosting environments. 
+
+## System overview
+
+Dapr consists of a data plane, the sidecar that runs next to your app, and a control plane that configures the sidecars and provides capabilities such as cert and identity management.
+
+### Self-hosted components
+
+* Sidecar (data plane)
+* Sentry (optional, control plane)
+* Placement (optional, control plane)
+
+For more information see [overview of Dapr in self-hosted mode]({{< ref self-hosted-overview.md >}}).
+
+### Kubernetes components
+
+* Sidecar (data plane)
+* Sentry (optional, control plane)
+* Placement (optional, control planee)
+* Operator (control plane)
+* Sidecar Injector (control plane)
+
+For more information see [overview of Dapr on Kubernetes]({{< ref kubernetes-overview.md >}}).
+
+## Performance summary for Dapr v0.11.3
```

Comment:
Given that we are close to v1.0-rc1, can we simply say these are v1.0-rc1 numbers?

Reply:
Another comment. Do you want to provide a link to the perf video from the community call in this article?

Reply:
Another Comment. You should mention HA mode for the control plane and give the numbers when in HA mode, since this will be the most common mode when deployed into K8s. Currently these numbers are too low on usage in non-HA mode

Reply:
No, because once this is merged it will appear on the docs website under 0.11.3, so it should be kept 0.11.3. Once we release we will update the version.

I do not want to provide a link to the youtube video, as this doc will change over time and the youtube video shows a point in time data and will quickly become outdated.

I added the HA numbers.